transport.go

```diff
@@ -294,7 +295,6 @@ func (t *Transport) Close() error {
 }
 
 func (t *Transport) closeServer() {
-	t.handlerMap.CloseServer()
 	t.mutex.Lock()
 	t.server = nil
 	if t.isSingleUse {
```

Comment:
On line 304, we will end up Closing the underlying udp connection if t.createdConn was true, this will end up closing the established quic connections. Do we want this?

Reply:
I don't see any other way, unless we want to keep the Go routine reading from the socket running.

Reply:
Otherwise, we'd have to keep a reference counter to eventually stop listening once all connections are closed. That would actually be quite nice.